app/components/post_list/post/body/acknowledgements/index.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+import compose from 'lodash/fp/compose';
+
+import {observeCurrentUser} from '@queries/servers/user';
+
+import Acknowledgements from './acknowledgements';
+
+import type {WithDatabaseArgs} from '@typings/database/database';
+
+const enhanced = withObservables([], (ownProps: WithDatabaseArgs) => {
+    const database = ownProps.database;
+    const currentUser = observeCurrentUser(database);
```

Comment:
I see that the component only uses the `id` and the `timezone` of the current User, so I wonder if is actually needed to pass the entire object instead of the needed props to avoid re-renders